scripts/create_transifex_resources.sh

```diff
@@ -1,67 +1,57 @@
 #!/bin/bash
 
-# This script is used to register InaSAFE translatable resources with Transifex
-# http://transifex.com
+# This script is used to register QGIS-Documentation translatable resources with Transifex
+# https://www.transifex.com
 #
 # Note that this script updates or creates entries in .tx/config file
+# so that they match expected formatting of tx-github integration slugs.
+# Used to first "tx push" existing translations to Transifex when enabling the github integration
 #
 # Tim Sutton, March 2013
+# Update: Harrissou Sant-anna, December 2020
 
-#
-# Sphinx documentation first
-#
+TARGETBRANCH='master'
```

Comment:
reminder: test whether this allows for an automatic value
```suggestion
TARGETBRANCH=`echo git branch | sed -n '/\* /s///p'`
```